plugins/compute/config/policy.json

```diff
@@ -9,6 +9,7 @@
   "context_is_compute_viewer":  "rule:context_is_compute_editor or rule:compute_viewer",
 
   "context_is_network_admin": "role:cloud_network_admin or role:network_admin",
+  "context_is_securitygroup_admin": "role:securitygroup_admin",
```

Comment:
Remove this rule completely

Reply:
Then undo all changes to the networking policy. There won't be any more name clashes